src/Tickets/Commerce/Gateways/PayPal/On_Boarding_Redirect_Handler.php

```diff
@@ -16,6 +16,32 @@
  *
  */
 class On_Boarding_Redirect_Handler {
+	/**
+	 * Return on boarding trouble notice.
+	 *
+	 * @TODO this method needs to be completely refactored into an admin page action.
+	 *
+	 * @since 5.1.6
+	 */
+	public function on_boarding_trouble_notice() {
+
+		$action_list = sprintf(
+			'<ol><li>%1$s</li><li>%2$s</li><li>%3$s %4$s</li></ol>',
+			esc_html__( 'Make sure to complete the entire PayPal process. Do not close the window you have finished the process.', 'event-tickets' ),
+			esc_html__( 'The last screen of the PayPal connect process includes a button to be sent back to your site. It is important you click this and do not close the window yourself.', 'event-tickets' ),
+			esc_html__( 'If you’re still having problems connecting:', 'event-tickets' ),
+			$this->settings->get_guidance_html()
+		);
+
+		$standard_error = sprintf(
+			'<div id="give-paypal-onboarding-trouble-notice" class="tribe-common-a11y-hidden"><p class="error-message">%1$s</p><p>%2$s</p></div>',
```

Comment:
Let's remove the Give reference.

Reply:
None of the methods on this file will stay. They are there just for reference. Error handling examples.